schema/poll.js

```diff
@@ -0,0 +1,28 @@
+const ssbSchemaDefintions = require('../v1/lib/ssbSchemaDefintions')
+
+const schema = {
+  $schema: 'http://json-schema.org/schema#',
+  type: 'object',
+  required: ['type', 'details', 'title', 'closesAt', 'version'],
```

Comment:
cosmetic :kiss: I like properties listed in the order they were mentioned in the required property. I think it's easier to read. superficial though